priv/perf/apps/load_test/lib/application.ex

```diff
@@ -15,10 +19,38 @@ defmodule LoadTest.Application do
         max_connections: max_connections
       )
 
+    :ok = start_services()
+
     {:ok, self()}
```

Comment:
Not sure why you're returning the Application pid here, you could have returned the started supervisor.

Reply:
aha!
now that we've removed that, you can add 
```
{:ok, _} = LoadTest.Service.Faucet.start_link(faucet_config)
```
below a supervisor, with :temporary restart strategy. 
```
:temporary - the child process is never restarted, regardless of the supervision strategy: any termination (even abnormal) is considered successful.


```